src/phaser/preload-scene.js

```diff
@@ -56,14 +56,22 @@ class PreloadScene extends Phaser.Scene {
             frameWidth: 32,
             frameHeight: 32,
         });
+  
+        this.load.image('obsticle-0', 'assets/images/enemy_litecoin.png');
+        this.load.image('obsticle-1', 'assets/images/enemy_usdcoin.png');
+        this.load.image('obsticle-2', 'assets/images/enemy_tetherer.png');
+        this.load.image('obsticle-3', 'assets/images/enemy_litecoin.png');
+        this.load.image('obsticle-4', 'assets/images/enemy_ethereum.png');
+        this.load.image('obsticle-5', 'assets/images/enemy_bitcoin.png');
+        this.load.image('obsticle-6', 'assets/images/enemy_tether.png');
 
-        this.load.image('obsticle-0', 'assets/images/litecoin.png');
-        this.load.image('obsticle-1', 'assets/images/usdcoin.png');
-        this.load.image('obsticle-2', 'assets/images/tethererc20.png');
-        this.load.image('obsticle-3', 'assets/images/litecoin.png');
-        this.load.image('obsticle-4', 'assets/images/ethereum.png');
-        this.load.image('obsticle-5', 'assets/images/bitcoin.png');
-        this.load.image('obsticle-6', 'assets/images/tether.png');
+        this.load.image('enemy_fly_coin-0', 'assets/images/enemy_coin_0.png');
```

Comment:
Do we need these assets? I don't think we use `enemy_fly_coin` anywhere?

Reply:
can we use enemy_fly_coin assets instead of just enemy_coin assets as they look more attractive

Reply:
I am working on it to add it instead of birds

Reply:
instead of the bird one I added a doge coin.

Reply:
since vijayasree is working on flying birds, I will remove these